doc/neps/nep-0046-sponsorship-guidelines.rst

```diff
@@ -86,7 +86,7 @@ Sponsors will get acknowledged through:
 - prominent logo placement on https://numpy.org/about/
 - logos displayed in talks about NumPy by maintainers
 - announcements of the sponsorship on the NumPy mailing list and the numpy-team
-  Twitter account
+  X account on https://x.com/numpy_team/
```

Comment:
I think I would prefer to not commit to publishing on any social media, i.e.
```
- announcements of the sponsorship on the NumPy mailing list.
```

Reply:
@star1327p Please delete this bullet point to finalize the PR.

Reply:
@InessaPawson Just made the change. Thanks for reviewing!